packages/ui-components/src/components/top-bar.ts

```diff
@@ -18,9 +18,14 @@ export class TopBar extends BaseElement {
     return html` <div
       class="text-white items-center bg-neutral-800 h-16 p-2 text-l flex flex-row"
     >
-      ${this.getLogo()}
+      ${this.logo && html`<img class="p-2 h-12" src="${this.logo}" />`}
```

Comment:
You could consider using lit's [`nothing`](https://lit.dev/docs/templates/conditionals/#conditionally-rendering-nothing) sentinel value, here.

```suggestion
      ${!!this.logo ? html`<img class="p-2 h-12" src="${this.logo}" />` : nothing}
```

Reply:
I would personally only do this if the check gets more complex. For example actually check if the logo is an empty string. Stuff like that.